examples/v1alpha3/metrics-with-timestamp-example.yaml

```diff
@@ -0,0 +1,53 @@
+apiVersion: "kubeflow.org/v1alpha3"
+kind: Experiment
+metadata:
+  namespace: kubeflow
+  name: metrics-with-timestamp
+spec:
+  objective:
+    type: maximize
+    goal: 0.99
+    objectiveMetricName: accuracy
+    additionalMetricNames:
+    - loss
+  algorithm:
+    algorithmName: random
+  parallelTrialCount: 3
+  maxTrialCount: 12
+  maxFailedTrialCount: 3
+  parameters:
+    - name: --lr
+      parameterType: double
+      feasibleSpace:
+        min: "0.01"
+        max: "0.03"
+    - name: --momentum
+      parameterType: double
+      feasibleSpace:
+        min: "0.3"
+        max: "0.7"
+  trialTemplate:
+    goTemplate:
+        rawTemplate: |-
+          apiVersion: batch/v1
+          kind: Job
+          metadata:
+            name: {{.Trial}}
+            namespace: {{.NameSpace}}
+          spec:
+            template:
+              spec:
+                containers:
+                - name: {{.Trial}}
+                  image: docker.io/andreyvelichkevich/timestamp-metric
```

Comment:
I had added you into kubeflowkatib, you can now push kubeflowkatib/timestamp-metric now
BTW, I think you can rename the image name to `kubeflowkatib/mxnet-mnist-example` to make all example has timestamp.

Reply:
I suggest to rename folder and file name also to `metrics-with-timestamp` to `mxnet-mnist-example`. There is no need  to highlight the term`with-timestamp`

Reply:
This is not a `mxnet-mnist-example`. 
I took code of pytorch-mnist from here: https://github.com/kubeflow/katib/blob/master/examples/v1alpha3/file-metrics-collector/mnist.py. 

Should I change example to Mxnet in that case? Mxnet example is that one: https://github.com/apache/incubator-mxnet/blob/master/example/image-classification/train_mnist.py.
Or we can name this example `kubeflowkatib/pytorch-mnist-example`?

Reply:
I missed it. IIRC @richardsliu created this image to show mxnet support.

Reply:
@johnugeorge So what can we do with this example?

Reply:
@richardsliu Can you point out at the Dockerfile used?